rplugin/python3/molten/images.py

```diff
@@ -199,11 +199,68 @@ def remove_image(self, identifier: str) -> None:
         self.to_make_invisible.add(identifier)
 
 
+class WeztermCanvas(Canvas):
+    """A canvas for using Wezterm's imgcat functionality to render images/plots"""
+
+    nvim: Nvim
+
+    def __init__(self, nvim: Nvim):
```

Comment:
As for getting options into this class, you can add it as a param here, and then just pass it along when the class is instantiated. You might need to add it as a param to all the Canvas classes, I'm not sure how strict python is with that.